docs/source/examples/BSTLD.rst

```diff
@@ -1,288 +1,183 @@
-########################
- Image with Box2D Label
-########################
+########
+ BSTLD
+########
 
-:ref:`supported_label_types:Box2D` is a kind of label type. It represents the 2D Box labels on
-images. (:numref:`Fig. %s <example-Box2D>`)
+This topic describes how to manage the "BSTLD" dataset.
 
-.. _example-Box2D:
+"BSTLD" is a dataset with :ref:`reference/label_format:Box2D` label type (:numref:`Fig. %s <example-bstld>`).
+See `this page <https://www.graviti.cn/open-datasets/BSTLD>`_  for more details about this dataset.
+
+.. _example-bstld:
 
 .. figure:: ../images/example-Box2D.png
    :scale: 50 %
    :align: center
 
-   The preview of an image with Box2D annotations.
+   The preview of a cropped image with labels from "BSTLD".
 
-*************
- Preparation
-*************
+**************************
+ Authorize a client object
+**************************
 
 First of all, create a :ref:`features/tensorbay_client:GAS Client`.
 
-.. code:: python
-
-    >>> from graviti import GAS
-
-    >>> ACCESS_KEY = "Accesskey-*****"
-    >>> gas = GAS(ACCESS_KEY)
-
-*****************************
- Read Dataset from TensorBay
-*****************************
-
-As mentioned in :ref:`quick_start:Quick Start`, obtain_ and fork_ should be done before reading a
-dataset from TensorBay. Then, pass the correct dataset name to the GAS client, and you will get a
-:ref:`features/tensorbay_client:Dataset Client`.
-
-Here, we take `BSTLD`_ as an example.
-
-.. _fork: https://docs.graviti.cn/guide/opendataset/fork
-
-.. _BSTLD: https://www.graviti.cn/open-datasets/BSTLD
-
-.. _obtain: https://docs.graviti.cn/guide/opendataset/get
-
-.. code:: python
-
-   >>> dataset_client = gas.get_dataset("BSTLD")
-
-If you are not sure about the dataset name, you can visit our `Opendataset Platform`_ to check all
-"forkable" open datasets.
-
-.. _opendataset platform: https://www.graviti.cn/open-datasets
-
-You can use the list method to print all your forked open datasets.
-
-.. code:: python
-
-    >>> list(gas.list_dataset_names())
-    ['Head Pose Image', 'TLR', 'BSTLD']
-
-In :ref:`basic_concepts:Dataset` ``BSTLD``, there are three
-:ref:`Segments <basic_concepts:Segment>`: ``train``, ``test`` and ``additional``,
-you can get the segment names by list them all.
-
-.. code::
-
-    >>> list(dataset_client.list_segment_names())
-    ['additional', 'test', 'train']
-
-You can get a segment by passing the required segment name.
-
-.. code:: python
-
-    >>> from tensorbay.dataset import Segment
-    >>> train_segment = Segment("train", dataset_client)
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 15-18
```

Comment:
See examples/getting_started_with_tensorbay.py for using start-after/end-before to include code block.